issues-guide.md

```diff
@@ -0,0 +1,100 @@
+Issues Guide
+=============
+This is a guide for collaborators and non-collaborators on how Github issues
+are handled and processed. 
+
+
+The Sprint
+-----------
+We are on a weekly sprint cycle and every Monday we get together for a meeting
+on Google Hangouts. Currently our meetings are at 12pm PDT / 7pm UTC and the 
```

Comment:
Rather than repeat information, I'll just link to that doc. Thanks for noticing that